recipes-core/packagegroups/packagegroup-ni-runmode.bb

```diff
@@ -23,6 +23,7 @@ RDEPENDS_${PN} = "\
 	librtpi \
 	linux-firmware-radeon \
 	lldpd \
+	niconfigpersistentlogs \
```

Comment:
Can you add a hyphen here, like `ni-configpersistentlogs`? That's the new package naming style for NI packages.

Reply:
Done.